ethcore/Cargo.toml

```diff
@@ -80,6 +80,13 @@ tempdir = "0.3"
 trie-standardmap = { git = "https://github.com/paritytech/parity-common" }
 
 [features]
+parity = ["work-notify", "price-info", "stratum"]
```

Comment:
Do we really need this?
I feel like we would get a compilation error in Parity anyway if we forget to enable a feature.

Reply:
I think this is just more future-proof in case we extract more optional features, because it's self-contained within `ethcore`. You don't have to go to all other crates and add your own feature, which is error-prone.
That would be especially important for things that might be disabled by default if the feature is not enabled.

Obviously can revert if you don't think it's necessary.

Reply:
It's not very proper, but it's fine I think.